package dmRender;

message Font
{
    message Glyph
    {
        required uint32 X = 1;
        required uint32 Y = 2;
        required uint32 Width = 3;
        required uint32 Height = 4;
    }

    repeated Glyph Glyphs = 1;
    required uint32 ImageWidth = 3;
    required uint32 ImageHeight = 4;
    required bytes ImageData = 2;
    required string VertexProgram = 5;
    required string FragmentProgram = 6;
}
